tests/methods/avas-6/input.dat

```diff
@@ -1,12 +1,12 @@
 #! Test AO Subspace regular expression on 3d orbitals
 
-refscf = -1646.62615747649124 
+refscf = -1646.6261575497
```

Comment:
Why are these reference values changing?

Reply:
Good question. The first one is pure SCF from Psi4.